src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -808,18 +808,23 @@ internal DefinitionResult GetDefinition(TextDocumentPosition textDocumentPositio
             {
                 try
                 {
+                    // Handle tokenizing for Peek Definition 
+                    Position tokenPosition = new Position();
```

Comment:
I'm suggesting changing the PeekDefinition class to use BufferPosition, since the pattern we have is that this uses Position's values + 1. That way it's clear what the expected input is. Otherwise, if someone else uses this class they might happily send in an unmodified Position object.